### PR TITLE
Add triple-cheese 1.2.1.3898

### PR DESCRIPTION
**Pull request by hannesbraun**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Review by vitorgalvao — CHANGES_REQUESTED** [review]
Please follow the same patter as https://github.com/Homebrew/homebrew-cask/pull/66765.

File: Casks/triple-cheese.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+cask 'triple-cheese' do
+  version '1.2.1.3898'
+  sha256 'd9a3e5f3e801513a74ec725bfcb722c0df3e96557a6be8813479f9ee73b59f8f'
+
+  # uhedownloads-heckmannaudiogmb.netdna-ssl.com was verified as official when first introduced to the cask
+  url "https://uhedownloads-heckmannaudiogmb.netdna-ssl.com/releases/TripleCheese_#{version.major_minor_patch.dots_to_underscores}_Mac.zip"
+  name 'Triple Cheese'
+  homepage 'https://u-he.com/products/triplecheese/'
+
+  pkg "TripleCheese#{version.split('.')[3]}Mac/TripleCheese #{version} Installer.pkg"
```
Comment (vitorgalvao) [inline Casks/triple-cheese.rb]: Does the installed app have spaces in the name?

Comment (hannesbraun) [reply Casks/triple-cheese.rb]: When installed, the Audio Unit is named "TripleCheese". But the homepage uses the version with a space in the name, that's why I chose that.

Comment (vitorgalvao) [reply Casks/triple-cheese.rb]: > But the homepage uses the version with a space in the name, that's why I chose that.

The [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md) is explicit. What matters it the name of the installed software, not what they call it on the page.

**Review by vitorgalvao — APPROVED** [review]

File: Casks/triple-cheese.rb (hunk 1 above)
Comment (hannesbraun) [reply Casks/triple-cheese.rb]: Okay, thank you :)

**Pull request merged**